### ISSUE TITLE
Wrong application name shown during install

### ISSUE THREAD

**Issue by MattBDev** [microsoft/winget-cli#722]
# Brief description of your issue
I needed to install a specific version of AdoptOpenJDK so I used the `--version` option to select version 11.0.10. During the install process it displayed "Found AdoptOpenJDK 15.0.1+9 (x64) [AdoptOpenJDK.OpenJDK]" which was wrong. It installed the right version of AdoptOpenJDK but during the install process it said that the name of the application was "AdoptOpenJDK 15.0.1+9 (x64)". The name of the package was 
# Steps to reproduce
First, show the package for AdoptOpenJDK using `winget show AdoptOpenJDK.OpenJDK`. All the details will show up for the newest version which at the time is 15.0.1.9. Then, run `winget show --versions AdoptOpenJDK.OpenJDK` and version `11.0.10` should appear on that list. Next, install that specific version of AdoptOpenJDK using `winget install --version 11.0.10 AdoptOpenJDK.OpenJDK`. The first line of the output is wrong and should display "Found AdoptOpenJDK 11.0.10+9 (x64) [AdoptOpenJDK.OpenJDK]".

# Expected behavior
I expected the output to say "Found AdoptOpenJDK 11.0.10+9 (x64) [AdoptOpenJDK.OpenJDK]" during the install process since that is the name found in the [package manifest for that version](https://github.com/microsoft/winget-pkgs/blob/master/manifests/AdoptOpenJDK/OpenJDK/11.0.10.yaml). 

# Actual behavior
Installing a package such as AdoptOpenJDK using the `--version` option shows the wrong application name at the beginning of the install process. 

# Environment
Edition	Windows 10 Pro
Version	20H2
Installed on	‎5/‎29/‎2020
OS build	19042.746
Experience	Windows Feature Experience Pack 120.2212.551.0

```
Windows Package Manager v0.2.3162 Preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19042.746
Package: Microsoft.DesktopAppInstaller v1.11.3162.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
--------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
```

**Comment by JohnMcPMS** [comment]
I believe that this was incidentally fixed by #767 as part of a refactor of the code involved.  [That change was first released in version v-0.2.10771-preview.](https://github.com/microsoft/winget-cli/releases/tag/v-0.2.10771-preview)

I was able to run the following command to demonstrate the fix:
```
> winget install AdoptOpenJDK.OpenJDK.15 --version 15.0.1.9
Found AdoptOpenJDK 15.0.1+9 (x64) [AdoptOpenJDK.OpenJDK.15]
```
whereas the latest version at the time is:
```
> winget show AdoptOpenJDK.OpenJDK.15
Found AdoptOpenJDK 15.0.2+7 (x64) [AdoptOpenJDK.OpenJDK.15]
```

Please let us know if the issue is still occurring.

**Comment by MattBDev** [comment]
@JohnMcPMS You can go ahead and close this issue as fixed. Although, I'm not sure if #767 fixed it or if it was a result of the changes made from microsoft/winget-pkgs#8670
